app/wasm/msg/types.go

```diff
@@ -0,0 +1,60 @@
+package msg
+
+import (
+	lvtypes "github.com/umee-network/umee/v4/x/leverage/types"
+)
+
+// AssignedMsg defines the msg to be called.
+type AssignedMsg uint16
+
+const (
+	// AssignedMsgSupply represents the call to supply coins to the capital facility.
+	AssignedMsgSupply AssignedMsg = iota + 0
```

Comment:
+0 or +1 here?

Reply:
The value will be same as umee-cosmwasm types https://github.com/umee-network/umee-cosmwasm/blob/987e4046360f4449e239c84eff7692bdd994e9cb/packages/cw-umee-types/src/msg_leverage.rs#L8